test/test.ts

```diff
@@ -1,13 +1,34 @@
 import { expect } from 'chai'
 import 'mocha'
-
-/*tslint:disable*/
-const fixtureJson = require('./fixture.json')
-const outputJson = require('./out/api_data.json')
-/*tslint:enable*/
-
+import * as apidoc from 'apidoc'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+const tests = [
+  {
+    only: false,
+    name: 'test1'
+  },
+  {
+    skip: true,
+    name: 'array'
+  },
+  {
+    skip: true,
+    name: 'array-as-types'
+  },
+  {
+    name: 'array-as-properties'
+  }
+]
 describe('Apidoc TS Plugin', () => {
-  it('Should include TS info in compiled api docs', () => {
-    expect(outputJson).to.deep.equal(fixtureJson)
+  tests.forEach(function (test) {
+    (test.only ? it.only : (test.skip ? it.skip : it))(test.name, async function () {
+      const dest = `out/${test.name}`
+      apidoc.createDoc({
+        src: `${test.name}`, debug: false, dest })
```

Comment:
This is weirdly indented

Reply:
I don't really see a problem (nor the linter) but changed